From 40aa4a6ae1531b189a25d373d642f5c7a1f7023d Mon Sep 17 00:00:00 2001
From: Andriy Grytsenko <andrej@rep.kiev.ua>
Date: Thu, 15 Nov 2012 05:50:44 +0200
Subject: [PATCH 10/22] [#3584798]We shouldn't disable DnD on non-native
 targets completely.

The handling dropping on non-native target should have some check
but not denial as it was before. Let user get error message if
dropping failed for now and do some other checks later.
---
 src/gtk/fm-dnd-dest.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/gtk/fm-dnd-dest.c b/src/gtk/fm-dnd-dest.c
index 21cf2c0..371e5f6 100644
--- a/src/gtk/fm-dnd-dest.c
+++ b/src/gtk/fm-dnd-dest.c
@@ -811,12 +811,16 @@ query_sources:
             else /* files inside trash are read only */
                 action = 0;
         }
+#if 0
+        /* bug #3584798: DnD(copy/move) to remote host(SFTP://) does not work.
+           we should do some check if target FS is R/O instead of this */
         else if(!fm_path_is_native(dest_path))
         {
             /* computer:/// and network:/// shouldn't received dropped files. */
             /* FIXME: some special handling can be done with menu:// */
             action = 0;
         }
+#endif
         else /* dest is a ordinary path */
         {
             /* determine if the dragged files are on the same device as destination file */
-- 
1.8.0.1

